test/Microsoft.AspNetCore.Mvc.Razor.Host.Test/TestFiles/Input/MultipleModels.cshtml

```diff
@@ -1,2 +1,2 @@
-﻿@model ThisShouldNotBeGenerated
+﻿@model ThisShouldBeGenerated
 @model System.Collections.IEnumerable
```

Comment:
I think we only changed the design time behavior so that we can provide coloring for all the model directives